### ISSUE TITLE
macOS help

### ISSUE THREAD

**Issue by dnisbet** [UoMResearchIT/r-tidyverse-intro#88]
https://cran.r-project.org/doc/manuals/r-release/R-admin.html#macOS

"To build R you need Apple’s ‘Command Line Tools’: these can be (re-)installed by xcode-select --install. (If you have a fresh OS installation, running e.g. make in a terminal will offer the installation of the command-line tools. If you have installed Xcode, this provides the command-line tools. The tools will need to be reinstalled when macOS is upgraded, as upgrading partially removes them.)"

"An X sub-system is required unless configuring using --without-x: see https://xquartz.macosforge.org/. R’s configure script can be told to look for X11 in XQuartz’s main location of /opt/X11, e.g. by

--x-includes=/opt/X11/include --x-libraries=/opt/X11/lib
although linked versions under /usr/X11 will be found."

So basically, if you're trying to get tidyverse onto a macOS install of Rstudio, open up a terminal and type in `make`.
For today's course however, when I tried `xcode-select --install` nothing happened (software installer came up, but wasn't doing anything). I ended up going to the Mac App Store and installing Xcode, which seems a little over the top but it worked...
